joss/paper.md

```diff
@@ -58,22 +58,25 @@ are generally organized in our companion package, `xskillscore`.
 # Statement of Need
 While other climate verification packages exist (_e.g._, `s2dverification`
 [@Manubens:2018] written in R and `MurCSS` [@Illing:2014] written with python-based
-`CDO`-bindings [@CDO]), `climpred` is unique for many reasons. (1) It spans broad
-temporal scales of prediction, supporting the weather, subseasonal-to-seasonal (S2S),
-and seasonal-to-decadal (S2D) communities. (2) `climpred` supports `dask`
-[@dask; @Rocklin:2015] and thus works across all computational scales, from personal
-laptops to supercomputers (HPC). This leads to
-verification of a global 5° x 5° resolution climate prediction in a few seconds,
-compared to the 8 minutes required by `MurCSS`. This allows for a truly interactive
-analysis experience. However, note that benchmarking is inherently biased and `MurCSS`
-is valuable for their rigorous replication of decadal climate prediction metrics.
```

Comment:
murccs is still valueable especially because it is a one-liner from end to end, whereas our oneline is just for the verify call. it just didnt fit the sentence as before.